packages/design-system/src/components/input/Input.stories.tsx

```
@@ -0,0 +1,69 @@
+  args: { ...Default.args },
+};
+
+export const Focus: Story = {
+  args: { ...Default.args },
+};
+
+export const Typing: Story = {
+  args: { ...Default.args, value: 'Text' },
+};
+
+export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)